services/core-web/src/components/noticeOfWork/NOWActionWrapper.js

```diff
@@ -30,6 +30,7 @@ const propTypes = {
   location: PropTypes.shape({
     pathname: PropTypes.string,
   }).isRequired,
+  isDisabledReviewButtons: PropTypes.bool,
 };
```

Comment:
Oh one more thing: 'isDisabledReviewButton' is the prop being passed in, but is defined as 'isDisabledReviewButtons' - can you update the propTypes and defaultProps to be the correct prop name?

Reply:
Good catch :)